test/jdk/sun/security/krb5/auto/NewSalt.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 6960894 8194486
+ * @bug 6960894 8194486 8139348
```

Comment:
I don't know if there are any rules or best practices about this, but I usually don't put a bugid on a test if it isn't specifically testing what this bug is about.

Reply:
OK, I'll remove it.